encfs/RawFileIO.cpp

```diff
@@ -220,14 +221,17 @@ bool RawFileIO::write(const IORequest &req) {
   ssize_t bytes = req.dataLen;
   off_t offset = req.offset;
 
+  int eno = 0;
   while ((bytes != 0) && retrys > 0) {
+    errno = 0;
```

Comment:
I just wondered why `write` would not have succeeded after having written something 10 times...
So I thought, as it did not manage to write what we want, perhaps it wrote only a small piece of our data and gently set `errno` to explain why.
Of course, not sure however it can happen...

Do you have a situation in mind where `write` could succeed 10 times without writing all the data ?